docs/start/qiskit-versioning.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Qiskit Versioning and Support Policy
+description: The versioning and support policy for Qiskit.
+
+---
+<span id="qiskit-version"></span>
+# Qiskit Versioning
+Qiskit version numbers follow [Semantic Versioning](https://semver.org/).
+The version number is comprised of 3 primary components, the major, minor, and
+patch versions. For a version number `X.Y.Z` where `X` is the major version,
+`Y` is the minor version, and `Z` is the patch version.
+
+Breaking API changes are reserved for major version releases. The **minimum**
+period between major version releases is one year. Minor versions will be
+periodically (currently every three months) published for the current major
+version which add new features and bug fixes. For the most recent minor version
+there will also be new patch versions published as bugs are identified and fixed
+on that release series. A tentative release schedule is included below:
+
+![timeline.png](../../public/images/timeline.png)
```

Comment:
Please put this at `/public/images/start/qiskit-versioning/timeline.png`. Also the image link should start with `/images`. 

I realize we don't document this in the README - I opened https://github.com/Qiskit/documentation/issues/367.

Reply:
Updated in: https://github.com/Qiskit/documentation/pull/366/commits/7d522c3392a86111f6b438ddfe15f1ec155342da